transport_plugins/bled112/iotile_transport_bled112/virtual_bled112.py

```diff
@@ -55,9 +56,15 @@ class BLED112VirtualInterface(VirtualIOTileInterface):
     HighspeedHandle = 21
     TracingHandle = 23
 
+    AdvertisingPacketVer = 1
+    json_args = None
+
     def __init__(self, args):
         super(BLED112VirtualInterface, self).__init__()
 
+        if args:
```

Comment:
Args is always passed as a dict (empty if no arguments) and will not ever be `None` (if it were None, line 69 would raise an exception).  It will not always be json though so the variable name `json_args` is misleading.  It is only json in one particular case where you invoke this class from the built-in `virtual-device` command line wrapper, but you're free to create this class programmatically yourself as well.